fastlane/Fastfile

```diff
@@ -237,7 +237,7 @@ platform :ios do
   lane :start_code_freeze do |options|
     ios_codefreeze_prechecks(options)
 
-    ios_bump_version_release(skip_deliver: true, skip_glotpress: true)
+    ios_bump_version_release
```

Comment:
@crazytonyli @mokagio JFYI: I had to make minor changes to the code freeze process.